modules/virt-uploading-image-web.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virtual_disks/virt-uploading-local-disk-images-web.adoc
+
+[id="virt-uploading-image-web_{context}"]
+= Uploading an image file using the web console
+
+Use the web console to upload an image file to a new Persistent Volume Claim (PVC).
+You can later use this PVC to attach the image to new virtual machines.
+
+.Prerequisites
+
+* You must have one of the following:
+** A raw virtual machine image file in either ISO or IMG format.
+** A virtual machine image file in QCOW2 format.
+* For best results, compress your image file according to the following guidelines before you upload it:
+** Compress a raw image file by using `xz` or `gzip`.
++
+[NOTE]
+====
+Using a compressed raw image file results in the most efficient upload.
+====
+** Compress a QCOW2 image file by using the method that is recommended for your client:
+*** If you use a Linux client, _sparsify_ the QCOW2 file by using the
+link:https://libguestfs.org/virt-sparsify.1.html[virt-sparsify] tool.
+*** If you use a Windows client, compress the QCOW2 file by using `xz` or `gzip`.
+
+.Procedure
+
+. From the side menu, click *Storage* -> *Persistent Volume Claims*.
```

Comment:
I know the module is for the web console, but just wondering if you should say "From the side menu of the web console" here?